Framework/PythonInterface/plugins/algorithms/AlignComponents.py

```diff
@@ -143,12 +157,18 @@ def PyInit(self):
                              doc="Maximum relative Z bound (m)")
         self.setPropertySettings("MaxZposition", condition)
 
-        # euler angles convention
+        properties = ["Xposition", "MinXposition", "MaxXposition", "Yposition", "MinYposition", "MaxYposition",
+                      "Zposition", "MinZposition", "MaxZposition"]
+        [self.setPropertyGroup(name, "Translation") for name in properties]
+
+
+        #
+        # Rotation Properties
         eulerConventions = ["ZXZ", "XYX", "YZY", "ZYZ", "XZX", "YXY", "XYZ", "YZX", "ZXY", "XZY", "ZYX", "YXZ"]
-        self.declareProperty(name="EulerConvention", defaultValue="YZX",
-                             validator=StringListValidator(eulerConventions),
-                             doc="Euler angles convention used when calculating and displaying angles,"
-                             "eg XYZ corresponding to alpha beta gamma.")
+        self.declareProperty(
+            name="EulerConvention", defaultValue="YZX", validator=StringListValidator(eulerConventions),
```

Comment:
What is the reason for using this particular version of Euler Angle as the default representation?

Reply:
I just go with the flow  
<img width="600" alt="Capture" src="https://user-images.githubusercontent.com/1136006/104246134-6df88180-5433-11eb-8a79-f4650ddc9e30.PNG">